packages/components/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 -   Replace deprecated wp.compose.withState with wp.element.useState. #8338
 -   Add missing dependencies. #8349
 -   Update all js packages with minor/patch version changes. #8392
+-   Refactor the `onFilterChange` method in the `AdvancedFilters` component. #8459
```

Comment:
Although I am not directly aware of plugins making use of Advanced filters, I do think this would be a breaking change and we should label it as such. 
Currently we just put it under `## Breaking changes`.
Given the changelog doesn't elude to how to fix this it might be worth adding a quick example of what change is required, something like `change: onFilterChange( 'value', rangeValue ); to onFilterChange( { property: 'value', value: rangeValue } );`

Reply:
@louwie17 Thanks for the review and suggestions!  Yes, we should put it under `## Breaking changes`. Updated in 99d45ea7f9d4c2658e0e0f79b9d004e0bdefd37a.